v-next/hardhat-utils/src/request.ts

```diff
@@ -125,7 +144,20 @@ export async function postJsonRequest(
       body: JSON.stringify(body),
     });
   } catch (e) {
-    ensureError(e);
+    ensureError<NodeJS.ErrnoException>(e);
```

Comment:
The error handling is the same in these functions: postJsonRequest, postFormRequest and download. Could it be moved into a function?